tests/v2/validation/provisioning/rke2/hostname_truncation.go

```diff
@@ -24,7 +25,7 @@ import (
 	capi "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
-func TestHostnameTruncation(t *testing.T, client *rancher.Client, provider Provider, machinePools []provv1.RKEMachinePool, defaultHostnameLengthLimit int, kubeVersion, cni string) {
+func TestHostnameTruncation(t *testing.T, client *rancher.Client, provider Provider, machinePools []provv1.RKEMachinePool, defaultHostnameLengthLimit int, kubeVersion, cni string, advancedOptions provisioning.AdvancedOptions) {
```

Comment:
Spoke offline about this, but we have a separate PR that will address this issue.